glossary.md

```diff
@@ -5,3 +5,66 @@ css_id: glossary
 
 # Glossary
 
+### Conformance terminology
+The keywords MUST, MUST NOT, REQUIRED, SHALL, SHALL NOT, SHOULD, SHOULD NOT, RECOMMENDED, MAY, and OPTIONAL in this document are to be interpreted as described in {{RFC2119}}.
```

Comment:
The link here need to be updated, or this line should be removed as well. Are these keywords actually relevant here?

Reply:
I'm going to remove them for now. They probably don't have the same significance here.